newa/__init__.py

```diff
@@ -1303,11 +1320,45 @@ def create_issue(self,
 
         try:
             jira_issue = self.connection.create_issue(data)
-            jira_issue.update(
-                fields={
-                    "labels": [
-                        *jira_issue.fields.labels,
-                        IssueHandler.newa_label]})
+            if fields is None:
+                fields = {}
+            # always add NEWA label to fields
+            if "Labels" in fields and isinstance(fields['Labels'], list):
+                fields['Labels'].append(IssueHandler.newa_label)
+            else:
+                fields['Labels'] = [IssueHandler.newa_label]
+                # IssueHandler.field_map["Pool Team"].id_: [{"value":
```

Comment:
Is this comment still needed in the final version?

Reply:
it is not